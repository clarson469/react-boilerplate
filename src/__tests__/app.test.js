import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';

import App from 'app';

describe('<App />', () => {
  test('Should render ok', () => {
    const wrapper = shallow(<App />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
