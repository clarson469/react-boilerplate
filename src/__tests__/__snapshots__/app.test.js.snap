// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Should render ok 1`] = `
<BrowserRouter>
  <main
    id="main-wrapper"
  >
    <HelloWorldContainer />
  </main>
</BrowserRouter>
`;
